dali/ft/daftformat.cpp

```diff
@@ -1002,79 +1005,64 @@ void CCsvQuickPartitioner::findSplitPoint(offset_t splitOffset, PartitionCursor
 {
     const byte *buffer = bufferBase();
     numInBuffer = bufferOffset = 0;
-    if (splitOffset != 0)
```

Comment:
git not making it easy to see the change, but I see it's just removal of this if and the else block.
tbh, I'm not familiar with this code, and am not sure what it was doing (or what it thought it was doing) in the case of splitOffset == 0 vs not.
@ghalliday - could you also review this please?

Reply:
I gave my explanation below why I removeed this condition and the related else branch.